.gitignore

```diff
@@ -6,14 +6,10 @@
 /build
 /dist
 /MANIFEST
-/setuptools-3.1-py2.7.egg
-/setuptools-3.1.zip
 /stone.egg-info
 .arcconfig
 .DS_Store
 .idea
-parser.out
```

Comment:
I think I remembered another detail. Did you run stone with `-v` or `-vv`? Depending on the level of verbosity I think it changes the debug flag passed into `ply`.